src/layout/components/ProfilePreview.vue

```diff
@@ -34,14 +37,25 @@ export default {
     avatar: {
       type: String,
       default: ''
+    },
+    fits: {
+      type: Array,
+      default: () => ['fill', 'contain', 'cover', 'none', 'scale-down']
     }
   },
   computed: {
+    userInfo() {
+      const name = this.$store.getters['user/userInfo'].name
+      if (this.isEmptyValue(name)) {
+        return ''
+      }
+      return name
+    },
     currentRole() {
       return this.$store.getters['user/getRole']
     },
     avatarResize() {
-      const defaultAvatar = 'https://avatars1.githubusercontent.com/u/1263359?s=200&v=4?imageView2/1/w/40/h/40'
+      const defaultAvatar = 'https://cube.elemecdn.com/3/7c/3ea6beec64369c2642b92c6726f1epng.png'
```

Comment:
Please use the user image getted from service: https://adempiere.github.io/proxy-adempiere-api/guide/default-modules/adempiere-api/user.html#get-api-user-info